flowey/flowey_lib_hvlite/src/_jobs/check_openvmm_hcl_size.rs

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! Compares the size of the OpenHCL binary in the current PR with the size of the binary from the last successful merge to main.
+
+use crate::artifact_openhcl_igvm_from_recipe_extras;
+use crate::artifact_openhcl_igvm_from_recipe_extras::OpenhclIgvmExtras;
+use crate::build_openhcl_igvm_from_recipe;
+use crate::build_openhcl_igvm_from_recipe::OpenhclIgvmRecipe::X64;
```

Comment:
having a bare `X64` in the global scope seems a bit wacky, lol. you should just import `OpenhclIgvmRecipe`, and then refer to `OpenhclIgvmRecipe::X64`